container/shared/src/main/filtered-resources/etc/startup.properties

```diff
@@ -10,7 +10,7 @@ mvn\:org.osgi/org.osgi.util.promise/1.1.1 = 9
 mvn\:org.apache.felix/org.apache.felix.coordinator/1.0.2 = 9
 mvn\:org.apache.felix/org.apache.felix.converter/1.0.14 = 9
 mvn\:org.osgi/org.osgi.util.function/1.1.0 = 9
-mvn\:org.apache.felix/org.apache.felix.configadmin/1.9.22 = 10
+mvn\:org.apache.felix/org.apache.felix.configadmin/${felixConfigadminVersion} = 10
```

Comment:
Can we add a note to `container/karaf/README.markdown` that this needs to be checked when Karaf is upgraded?

Reply:
done.